initiatives/2020-04-docs.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "type": "sourcecred/initiativeFile",
+    "version": "0.1.0"
+  },
+  {
+    "title": "Initial Documentation Push",
+    "timestampIso": "2020-04-01T10:00:00.000Z",
+    "weight": {
+      "incomplete": 200,
+      "complete": 1000
+    },
+    "completed": false,
+    "champions": [
+    ],
+    "dependencies": [
+    ],
+    "references": [
+      "https://github.com/sourcecred/docs/issues/5",
+      "https://discourse.sourcecred.io/t/creating-documentation-on-github-review-culture/582",
+      "https://discourse.sourcecred.io/t/documentation-glossary-and-wishlist/354",
+      "https://discourse.sourcecred.io/t/weekly-community-call-1-28-2020/605",
```

Comment:
While I appreciate the link to community call notes (which are generally documentation), presumably these are also going to be contributions to the community call Initiative I've been tasked to help flesh out? If so, I would say remove these. Call notes also just feel kinda like a different type of documentation...this feels more like an "official docs" initiative. 

More generally, while I feel I've made a few abstract, non-concrete docs contributions (e.g. playing docs "consultant" on calls, providing input to decisions), not sure how that would show up in this initiative. I have had the thought I would like to get more cred in the future by making my own PRs, which is a good thing IMO, as it shows the incentives are working. Alas, have been too busy to do anything other than review lately, which I presume will flow a bit of cred my way, via docs Initiative->PR->reviews/comments.

Reply:
I included them in references b.c. they were calls in which we discussed the documentation effort. But they definitely aren't as relevant as those posts/issues which were specifically oriented on the docs work. Removed per your suggestion.